lib/rucio/tests/test_bin_rucio.py

```diff
@@ -259,14 +264,16 @@ def test_upload_repeated_file(self):
         remove(tmp_file1)
         remove(tmp_file2)
         remove(tmp_file3)
-        nose.tools.assert_not_equal("File %s:%s successfully uploaded on the storage" % (self.user, tmp_file1[5:]) in out, None)
+        nose.tools.assert_true((self.upload_success_str % tmp_file1_name) in out)
```

Comment:
Why was this changed from assert_not_equal to assert_true? Did the logic change for this case? (IIUC trying to upload a file already in the catalog is not an error anymore?)

Reply:
No, probably a copy paste error. Thanks!

Reply:
But this was wrong anyway wasn't it? Assuming the `... in out` expression returns either True or False, it's always not equal to None.

Reply:
I agree that the old test logic didn't make sense because the test was always returning success. IIUC with the new logic, the test will return OK if the file is successfully uploaded after being previously removed - if this is the intended test behavior then it looks fine now.